CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [VM/Bulk migrate] Sort hosts by VM's pools [#4462](https://github.com/vatesfr/xen-orchestra/issues/4462) (PR [#5308](https://github.com/vatesfr/xen-orchestra/pull/5308))
```

Comment:
I don't think it's very clear what it does, from a user point of view. Suggestion:
"Sort hosts in the select so that the hosts on the same pool are shown first"